soh/soh/SohMenuBar.cpp

```diff
@@ -1467,7 +1469,8 @@ extern std::shared_ptr<ValueViewerWindow> mValueViewerWindow;
 void DrawDeveloperToolsMenu() {
     if (ImGui::BeginMenu("Developer Tools")) {
         ImGui::BeginDisabled(CVarGetInteger("gDisableChangingSettings", 0));
-
+        UIWidgets::EnhancementCheckbox("Stairs heap display", "gStairsDisplay");
+        UIWidgets::Tooltip("Visualises the stair heap");
```

Comment:
IIRC there was some talk about this being used for more than just the stairs outside shadow temple, so genericizing this language would be good (or do the other places where this is relevant require a separate visualizer?)